test/integration/contact.test.js

```diff
@@ -0,0 +1,31 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+import { data, wrongData } from '../fixtures/contact.us';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+
+
```

Comment:
Codacy found an issue: [More than 1 blank line not allowed.](https://app.codacy.com/gh/UptimaNGR/uptima-backend/pullRequest?prid=5963307)